frontend/server/src/ClaimRule.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types=1);
+namespace OmegaUp;
+
+class ClaimRule implements \ParagonIE\Paseto\ValidationRuleInterface {
+    /** @var string $failure */
+    protected $failure = 'OK';
+
+    /** @var string $rule */
+    protected $rule;
```

Comment:
```suggestion
    /** @var string */
    protected $claim;
```

también hay que cambiarlo en el constructor. esto es lo que está causando errores.

Reply:
Listo